lucet-runtime/lucet-runtime-internals/src/context/mod.rs

```diff
@@ -628,10 +628,12 @@ impl Context {
     ///
     /// TODO: the return type of this function should really be `Result<!, nix::Error>`, but using
     /// `!` as a type like that is currently experimental.
-    #[inline]
```

Comment:
the `inline` goes away because at first I thought "to do this it needs to not be inlined into the signal handler!!" but then scrapped the first very bad idea for only using a function pointer to `set_from_signal`. And if it's only used as a function pointer in the code, there's nowhere to inline, so even wanting to inline it would be misleading to the reader.

Reply:
Thanks for explaining this, makes sense to me. Nice catch!